v0/destinations/posthog/data/PHPropertiesConfig.json

```diff
@@ -140,5 +127,41 @@
     "destKey": "$app_build",
     "sourceKeys": "context.app.build",
     "isMobile": true
+  },
+  {
+    "destKey": "$viewport_height",
+    "sourceKeys": "context.viewport.height"
+  },
+  {
+    "destKey": "$viewport_width",
+    "sourceKeys": "context.viewport.width"
+  },
+  {
+    "destKey": "utm_source",
+    "sourceKeys": "context.campaign.source"
+  },
+  {
+    "destKey": "utm_medium",
+    "sourceKeys": "context.campaign.medium"
+  },
+  {
+    "destKey": "utm_campaign",
+    "sourceKeys": "context.campaign.campaign"
+  },
+  {
+    "destKey": "utm_content",
+    "sourceKeys": "context.campaign.content"
+  },
+  {
+    "destKey": "utm_term",
+    "sourceKeys": "context.campaign.term"
+  },
+  {
+    "destKey": "utm_name",
```

Comment:
@mihir-4116 can you please refer to the doc where this property name is mentioned? 
I am referring to:
1. https://github.com/PostHog/posthog/blob/0c65195a9d1f23690a64457130f4ca1709ded4fc/frontend/src/lib/components/PropertyKeyInfo.tsx#L327
2. https://posthog.com/manual/utm-segmentation

I do not see it being mentioned over there. Am I missing something?

Reply:
@shrouti1507 i was able to see this property in doc, but it's not showing anymore. i am removing it and update the doc card accordingly.